### ISSUE TITLE
dataset size and accuracy

### ISSUE THREAD

**Issue by rinoshinme** [xdw1995/Fall_Down#1]
How many videos do you have as the training data? And the final accuracy? Can you share the training video?

**Comment by xdw1995** [comment]
> How many videos do you have as the training data? And the final accuracy? Can you share the training video?
The format of the data can be referred to 'now_data_train.txt'.This is part of the data selected by random sampling at the same interval from subjcet1 to 13 during training in order to reduce overfitting.The total number of training sets is about 12*11*3*2, but there's some data that doesn't have a label.Our interval sampling training set is only one fourth of the total.
The accuracy reaches 88% for a single frame and 92% for multiple frames on our validation set.
Finally , the results of multiple models are fused, and the accuracy reaches 95% approximately.

All the data can be downloaded from https://sites.google.com/up.edu.mx/har-up manually as we did.
There are two reasons for this choice of data：
1.There are too many repetitive actions because we only chose 5 types of actions.
2.It takes too long to download one by hand.

**Comment by xdw1995** [comment]
> How many videos do you have as the training data? And the final accuracy? Can you share the training video?
> The format of the data can be referred to 'now_data_train.txt'.This is part of the data selected by random sampling at the same interval from subjcet1 to 13 during training in order to reduce overfitting.The total number of training sets is about 12113*2, but there's some data that doesn't have a label.Our interval sampling training set is only one fourth of the total.
> The accuracy reaches 88% for a single frame and 92% for multiple frames on our validation set.
> Finally , the results of multiple models are fused, and the accuracy reaches 95% approximately.
> 
> All the data can be downloaded from https://sites.google.com/up.edu.mx/har-up manually as we did.
> There are two reasons for this choice of data：
> 1.There are too many repetitive actions because we only chose 5 types of actions.
> 2.It takes too long to download one by hand.

The accuracy of all our models on the training set was about 98%.The average accuracy of multiple models fluctuates around 90%.My sense is that the accuracy should not differ much across all data sets.
